plugins/ccgx/ccgx.quirk

```diff
@@ -69,20 +70,33 @@ Plugin = ccgx
 GType = FuCcgxDmcDevice
 Summary = Dock Management Controller Device
 ParentGuid = USB\VID_03F0&PID_096B
+Vendor = HP
 Name = HP USB-C/A Universal Dock G2
 CcgxDmcTriggerCode = 0x01
 CcgxImageKind = dmc-composite
 InstallDuration = 180
 RemoveDelay = 162000
 
+# HP Thunderbolt Dock G4 Root Hub
+[USB\VID_1D5C&PID_5801]
+Summary = USB Hub
+Name = Thunderbolt Dock G4 top most USB Hub
```

Comment:
This should appear in list of devices as "HP Thunderbolt Dock G4". So this string needs to be modified.